### PR TITLE
Take connectors as a param on quote

### PR DESCRIPTION
**Pull request by vhpoet**
In some cases I might want to specify a list of connectors to get quotes from. I also need this working with the [plugin-virtual](https://github.com/interledger/js-ilp-plugin-virtual). [See this](https://github.com/interledger/js-ilp-plugin-virtual/blob/master/src/plugin/noob_plugin.js#L221-L225).

### CODE REVIEW

File: src/lib/client.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ class Client extends EventEmitter {
         destination_precision: params.destinationPrecision,
         destination_scale: params.destinationScale
       }
-      const connectors = yield plugin.getConnectors()
+      const connectors = params.connectors ? params.connectors : yield plugin.getConnectors()
```
Comment (sentientwaffle) [inline src/lib/client.js]: `params.connectors || yield ...` would be simpler.

**Comment by sentientwaffle** [comment]
Please update the function's comment to include the new parameter.

**Comment by emschwartz** [comment]
Ditto to @sentientwaffle's comments and please add a test for this

File: src/lib/client.js (hunk 1 above)
Comment (vhpoet) [inline src/lib/client.js]: @sentientwaffle that doesn't work :P

**Comment by vhpoet** [comment]
@sentientwaffle @emschwartz done

**Comment by emschwartz** [comment]
LGTM

**Pull request merged**